### DISCUSSION TITLE
👀 Discussions

### DISCUSSION THREAD

**Discussion by canisminor1990** [lobehub/sd-webui-lobe-theme#53] [General]

**Comment by canisminor1990** [comment]
Chat about anything and everything here

**Reply by soyatlas** [reply]
This theme is a porsche (kitchen) for a formula1 (a1111 webui)
Thank you.

Suggestion: apply a filter so that the checkpoints (lora, etc.) can have tags, for example #SD1.5 #SD2.1 #man #girl #nsfw #comic #photographic etc.

**Reply by Mafiav-afk** [reply]
Why isn't my embedded model displaying?

**Reply by yuhui03** [reply]
Suggestion: loar folder is best to correspond with the model, the folder is much better to find

**Reply by caojingfei** [reply]
双列模式下, 没有进度条也没有预览的bug

**Reply by jasoncow007** [reply]
双列模式下, 没有进度条也没有预览的bug，浮动失效bug，send outpaint失效bug，，，期待解决

**Reply by caojingfei** [reply]
双列模式下, 没有进度条也没有预览的bug, 解决方案: 关闭或卸载这个sd-webui-supermerger插件就好了

**Reply by Ataoya1** [reply]
生成按钮下面的快捷小按钮  自动跑到了主界面去了

**Comment by Dragoy** [comment]
I love the new interface, you are a genius!

**Comment by z44sm** [comment]
Good looking UI few issues put a bug report in there,  but overall better UI than original.  thanks for the project.

**Comment by Zullian** [comment]
I like the modern design, but atm it's missing settings, like colors, background image, scaling/reorganization of windows/areas.

**Comment by imnekosama** [comment]
Unable to view image information

**Comment by Moya207713** [comment]
1. 正负提示词区域建议支持自动高度, 而不是固定高度. 
2. Tag拆分的这个功能建议参考下https://moonvy.com/apps/ops/这个, 结合到一起就真的无敌.

**Reply by Moya207713** [reply]
3. 设置宽高批次,到设置随机种子这部分, 占用的空间太多了, 把下面的空间留给插件更好

**Reply by caojingfei** [reply]
双列模式下, 没有进度条也没有预览的bug，浮动失效bug，send outpaint失效bug，，，期待解决

**Comment by pessivesto** [comment]
your theme is awesome However, when using it on an iPad or iPhone, there is a problem with the top sidebar taking up the entire screen. If you press the top arrow button to switch tabs, then the bottom arrow button disappears and stays in the top bar menu forever. Your theme's menu sorting and prompt wizard features are really nice. I want to enjoy this benefit on mobile as well. Please I hope you can solve this problem.

please check this out screenshot link. (it's ipad 12.9)

http://m-odt.synology.me:5678/mo/sharing/tZq89YKRi

**Reply by caojingfei** [reply]
双列模式下, 没有进度条也没有预览的bug，浮动失效bug，send outpaint失效bug，，，期待解决

**Comment by Design-W** [comment]
右侧的lora预览希望可以选择扩展lora，因为使用additional networks 功能的LORA，在此处无法显示

**Reply by anne43983959** [reply]
> 右侧的lora预览希望可以选择扩展lora，因为使用additional networks 功能的LORA，在此处无法显示

建议放弃使用addnet，内置的LoRA，配合LoRA block weight，已经可以完全取代所有addnet的功能，而且现在LoCon和LyCORIS插件已经失去了对addnet的支持，它已经完成了推广LoRA的历史使命，可以光荣退役了，没有图形化界面，也不支持子目录分类，不是kitchen theme的问题，是addnet自己的问题

**Reply by caojingfei** [reply]
双列模式下, 没有进度条也没有预览的bug，浮动失效bug，send outpaint失效bug，，，期待解决

**Comment by canisminor1990** [comment]
![image](https://user-images.githubusercontent.com/17870709/236496873-c2a28059-f46c-447b-a15d-a7ef3917850f.png)
[✨](https://github.com/canisminor1990/sd-webui-kitchen-theme/commit/67accf3553db5721b9aca3516cdae6a05c92eeb7) [feat: t2i & i2i support draggable view](https://github.com/canisminor1990/sd-webui-kitchen-theme/commit/67accf3553db5721b9aca3516cdae6a05c92eeb7)

**Reply by anne43983959** [reply]
这个设计确实好，而且有些界面设计可以参考Cozy-Nest ( https://github.com/Nevysha/Cozy-Nest )，那边比较有优势的地方在于改动可以实时更新，但是附加网络面板还是一大堆bug，对Civitai Helper的兼容尤其差，这一点kitchen比Cozy-Nest强太多了
还有一点就是提示词框高度调整的问题，如果界面左右分离了，其实可以考虑放开调整高度了，不然使用latent couple的时候prompt一多就没法编辑了

**Reply by Mafiav-afk** [reply]
Why isn't my embedded model displaying?

**Reply by caojingfei** [reply]
双列模式下, 没有进度条也没有预览的bug，浮动失效bug，send outpaint失效bug，，，期待解决

**Comment by M30W-git** [comment]
Cool theme, but I would love to see the quick-settings back up (in the header).
Maybe a toggle option for it ?! or something like that, because if i disable the side bar, there is no more extra settings showing at all.

**Comment by PyrateGFXProductions** [comment]
Hey, I really like what you've done with the place. 👍

**Comment by EricRollei** [comment]
It's really nice, thank you!
If I could change the prompt window sizes that would be great addition.  If my prompt is too long, I can't see all of it and wish I could make the window bigger.

**Reply by canisminor1990** [reply]
> It's really nice, thank you!
> If I could change the prompt window sizes that would be great addition.  If my prompt is too long, I can't see all of it and wish I could make the window bigger. 

Try the latest version, add prompt textarea display mode in ⚙️ setting

**Reply by superw2018** [reply]
I rewrite the setting defalt config 
export const defaultSetting: WebuiSetting = {
  enableExtraNetworkSidebar: true,
  enableHighlight: false,
  enableSidebar: true,
  enableWebFont: true,
  extraNetworkCardSize: 86,
  extraNetworkFixedMode: 'fixed',
  extraNetworkSidebarExpand: true,
  extraNetworkSidebarWidth: 340,
  i18n: 'zh_CN',
  layoutHideFooter: false,
  layoutSplitPreview: false,
  liteAnimation: false,
  logoCustomTitle: 'OnlySd',
  logoCustomUrl: 

but it is not useful,how can i set the default setting when the page is loaded.

**Comment by zeno17z** [comment]
希望选项卡可以自己选择先后顺序，不然常用的可能总是被排在后面。还希望可以固定生成按钮和图片窗口，不然扩展插件很多在页面下方，总是不能实时看到生成效果

**Reply by caojingfei** [reply]
双列模式下, 没有进度条也没有预览的bug，浮动失效bug，send outpaint失效bug，，，期待解决

**Comment by M3g4m3** [comment]
can you upgrade a mobile UI :">

**Comment by MarshallAi** [comment]
我的界面模型那一列一直不显示模式了！

**Reply by MarshallAi** [reply]
我的界面模型那一列一直不显示所有的模型了。不知道怎么解决？

**Reply by caojingfei** [reply]
双列模式下, 没有进度条也没有预览的bug，浮动失效bug，send outpaint失效bug，，，期待解决

**Comment by MarshallAi** [comment]
我的界面模型那一列一直不显示所有的模型了。不知道怎么解决？

**Comment by todpeng777** [comment]
真是超棒的设计，原来的UI越来越复杂，真的需要修整一下了。我有个想法，要不要把采样方法和采样步骤也挪到左侧边栏去，而宽高度批次设置挪到画布下面去？好像这里不能添加图片附件，我P了一张草图

**Reply by todpeng777** [reply]
![111](https://github.com/canisminor1990/sd-webui-kitchen-theme/assets/125846611/daf0d2a7-cb18-4eb6-bf2d-1b16e0153238)

**Reply by todpeng777** [reply]
或者像前面一位的反馈一样，把生成按钮固定住？个人不成熟意见仅供参考，太需要你们大神的优秀插件了：）
![112](https://github.com/canisminor1990/sd-webui-kitchen-theme/assets/125846611/c682ab74-5fa5-466a-912f-3ce631986f97)

**Reply by M30W-git** [reply]
I really need this top-bar ! O_o'

**Comment by T3chRomanc3r** [comment]
Seems like panel with size slider in ExtraNetwork Sidebar can't be resized in both Edge and Chrome, but it can be resized in Firefox, because size arrow in right down corner is only visible in firefox
![missing_arrow](https://github.com/canisminor1990/sd-webui-kitchen-theme/assets/45420556/c27ee576-674d-4ce1-b8e9-2e7cc6db3339)

**Reply by soyatlas** [reply]
Thanks, switching to fire browser 👍

**Comment by Piscabo** [comment]
What is the possibility of implementing a search-on-page function (ctrl+F)? If I knew how I would implement it:
https://github.com/rakina/find-in-page-api/blob/master
Please!

**Comment by 10ywzheng** [comment]
大佬，我装了双语插件，导致这个位置的 按钮没有自适应高度，文字超出去了。
看了下代码，是不是简单删掉这行就行啦？（如图红色箭头）
帮忙修复下多谢感激 
![WechatIMG136](https://github.com/canisminor1990/sd-webui-kitchen-theme/assets/3376325/ea727d6e-b1c8-4235-bc48-d192adf1904b)

**Comment by canisminor1990** [comment]
🤯🎉 Bump v2.0.0 and rename to Lobe Theme, add several theme customize features, [more info](https://github.com/canisminor1990/sd-webui-lobe-theme)

**Comment by EricRollei** [comment]
Okay, I really hate that my Kitchen theme turned into Lobo without asking.  Scripts doesn't work as I can't get access to the full pick list.  I tried going back to kitchen in settings but still broken.

**Reply by canisminor1990** [reply]
fixed

**Reply by ridai260** [reply]
Same, how does one go back to Kitchen? My UI looks worse than yesterday, changed the 'logo' to kitchen but its still a full black theme with things missing.
EDIT: Ah moved to legacy, found it.

**Reply by canisminor1990** [reply]
The legacy version can be accessed at [sd-webui-kitchen-theme-legacy](https://github.com/canisminor1990/sd-webui-kitchen-theme-legacy)

**Reply by Piscabo** [reply]
The progress bar still disappears, when generation starts... 
![image](https://github.com/canisminor1990/sd-webui-lobe-theme/assets/123719842/177db105-1063-4a6b-97c0-915a807591c1)

**Reply by Piscabo** [reply]
Please Pin the top tabs of Extra Network. 
![image](https://github.com/canisminor1990/sd-webui-lobe-theme/assets/123719842/46f4cdfb-49f1-4e5a-8560-ce12357d360d)

**Reply by Piscabo** [reply]
Is there a fix in the css code where I can do this? It is not very intuitive to have tabs scroll up with the cards...

**Comment by sdwx27** [comment]
ADetailer插件模型列表显示不出来

**Comment by pto2k** [comment]
Hi, After the name changing update, there is this rectangle color box on the top-right corner . 
I think it was for the blurred background color. but now it is solid shape.
Any setting I can change to fix that?
Thanks

**Reply by suonnon** [reply]
I had the same issue with latest update.
Also another issue:
- When refreshing the "CheckPoint" tab, the tab becomes "loading" forever, sometimes it refreshes the tab but no checkpoints there.

I'm using A1111 v1.4.0 (latest update)

**Comment by steinhaug** [comment]
I really love your theme. I do have a bugfix though, and its for the Extra Networks. I cannot load the "data" page for the LORAs so I cannot see what words I need to add. This was in the form of a little ? in the upper right corner normally.

Also, where are the settigs saved by the way ?

**Reply by makisekurisu-jp** [reply]
I can't save the settings either.

**Comment by Piscabo** [comment]
More overlapping issues with ControlNet. 
![image](https://github.com/canisminor1990/sd-webui-lobe-theme/assets/123719842/3dfd4042-d6d3-4205-97cd-dd725342e041)

**Comment by laignguihua** [comment]
更新到2ef28fb3版本后的“提示词语法高亮”功能想法很好不过问题挺多的
1.白色主题下，打开“提示词语法高亮”后，选中的词的底色不显眼这个太影响使用，原来的黄色底就很醒目容易识别，由于提示词高亮显示后字体的颜色不再单一，可以考虑深颜色的选中单词的底色。黑色主题我没有用过不得而知。
2.与sd-webui-tagcomplete插件冲突，例如：在提示词输入框里键按m键，sd-webui-tagcomplete弹出选择菜单第一个是“masterpiece”但点击后只会输入“m”，奇怪的是在sd-webui-prompt-all-in-one插件的关键词输入窗口进行同样的操作是可以输入“masterpiece”的，不过输入的词不是绿色的，是黑色的。
希望作者可以修正一下着两个问题。

**Comment by DAQINCHA** [comment]
非常美丽的主题！！！但是不知道为什么，在mac的safari上非常非常非常的卡，希望可以解决～在Chrome和Edge上都没问题。

**Comment by Piscabo** [comment]
Text2Image scrolling between images not working.
![image](https://github.com/canisminor1990/sd-webui-lobe-theme/assets/123719842/e319929b-5674-46a0-97e0-be3d5244c705)

**Comment by makisekurisu-jp** [comment]
Restarting the webui settings cannot be saved.

**Comment by vrai17** [comment]
I cant use my mouse scroll now
Any fix?

**Comment by laignguihua** [comment]
“附加网络”右键没有办法调用lora-prompt-tool插件的菜单了！

**Reply by gumu1996** [reply]
我的也是，新版本用不了，你退回到467e09f这个版本就还可以用

**Comment by bgycn** [comment]
在双列模式下，快捷键Ctrl+Enter无效

**Comment by canisminor1990** [comment]
![](https://gw.alipayobjects.com/zos/kitchen/az49akOKJT/feat_pwa.webp)

#### Progress Web App

You can use Progressive Web Apps [PWA](https://support.google.com/chrome/answer/9658361) for a fast stable diffusion experience on your computer or mobile device.

- On your computer, open Chrome.
- Go to stable diffusion website you want to install
- At the top right of the address bar, click `Install`
- Follow the onscreen instructions to install the PWA



## [Version&nbsp;2.7.0](https://github.com/canisminor1990/sd-webui-lobe-theme/compare/v2.6.5...v2.7.0)
<sup>Released on **2023-07-05**</sup>


#### ✨ Features

- **misc**: Support `PWA` progressive web app.

**Comment by BaiterYamato** [comment]
is there any way to save the theme direct on sd and not on cache browser?

**Comment by izhy56103** [comment]
When I click the settings button, the browser page goes blank. My browser version is 116.0.1938.10  dev 64 ，What can I do？ Help me，pls

**Reply by izhy56103** [reply]
My browser version is Edge  116.0.1938.10 dev 64

**Comment by zaderei** [comment]
Finally found a theme that i like, but I have a question (and I feel super stupid rn) - when i close the quick settings tab and it disappears, where do I open it back? 🥲

**Comment by pto2k** [comment]
Feature Request: Could you please add shortcuts for open/close side panels?
Thank you!

**Comment by spirit9135** [comment]
The new version has a bug, and my settings have been restored to default. Resetting does not work either.

**Comment by Saradominist** [comment]
Can't open settings for theme

**Comment by sanliang233** [comment]
双列模式下文生图无法看到实时预览、进度条
图生图一切正常

**Comment by nkinfe** [comment]
文生图菜单栏，感觉用左右滑动来显示比三个小点点开下拉菜单"..."会方便很多

**Comment by xiaoyang1212** [comment]
求解答~
在这里可以训练模型吗？
创建了Embedding以后，显示文件在Created: /home/admin/stable-diffusion-webui/embeddings/aaaaa.pt，该去哪里找这个文件夹？本地文件夹里没有找到这个文件~

**Comment by laignguihua** [comment]
提个小建议
“附件网络侧边栏”默认的标签改成“基础模型”或记住上次的标签会更方便。毕竟一上来肯定是确定用那个基础模型的，或者默认是使用最频繁的Lora标签也总比embedding来的方便。

**Comment by kadevin** [comment]
Bug反馈：开启 “Prompt 语法高亮” 功能后，会导致在提示词输入框中，无法使用 “标签自动补全”插件的键盘按键选择候选词功能，希望修复，感谢！（webui和插件版本均为今天更新的最新版本）

**Reply by caojingfei** [reply]
双列模式下, 没有进度条也没有预览的bug，浮动失效bug，send outpaint失效bug，，，期待解决

**Comment by zywwdh** [comment]
发现一个小问题，比如说Lora模型文件夹里面建立了不同的子文件以后，预览点击完子文件夹，当返回查看其他目录时，上一个点击的路径没有清除，需要手动删除检索那里的路径

**Comment by bluelovers** [comment]
按下設定之後 畫面會變成空白

**Comment by bluelovers** [comment]
hope can always display `generate` button like `Cozy-Nest` do

**Comment by demi86** [comment]
In the WebUI, I can only scroll using the arrow keys, and unfortunately, a part of the screen is cut off at the top. It's a bit frustrating, especially because I've activated several plugins that are arranged in a long row. So, scrolling through them takes forever and feels like a never-ending task.

To make things worse, on the far right of the screen, there's a bar that's excessively long. It stretches way beyond what's necessary. And let's not forget about the settings window. It's quite large, displaying labels like "Process Queue" and "Clear Queue - Queue is empty." However, when I click on the "Settings" symbol, that same window pops up, and I can't seem to exit it. No matter what I try, whether it's using the mouse or keyboard shortcuts like "Esc," "CTRL & C," "CTRL & Q," "Enter," or "Del," I'm stuck. The only way out is to reload the URL or restart via the terminal and then comes the long loading time again to restart the WebUI. xD

I like the style of the new UI but at the moment It's an adventure navigating through this WebUI, with its quirks and unexpected challenges. 

Even here in the comment section, I have better options to resize the form field compared to the Web UI. It's quite ironic that a simple comment feature offers more flexibility than the Web UI. :-P

I've tested it with Google Chrome and with Mozilla Firefox and by both Browsers the same problem. 

Could it possibly be due to my monitor's high resolution of 3440 x 1440 pixels? This ultrawide format provides a large and immersive display area, but it can cause issues with certain applications. 

Downscaling the resolution isn't a good solution either, as it leads to a loss of clarity and sharpness in the display. Furthermore, it can cause additional issues with UI scaling and layout, as elements designed for a higher resolution may not display correctly at a lower resolution.

Or is here the lack of the ultrawide support? because this can be a common issue too, i had this experience with various user interfaces in the past. This happens, because not all applications and systems are designed to natively support ultrawide formats like 3440 x 1440 pixels.

**Comment by Tilmirs** [comment]
I hope the tab bar icon supports customization

**Comment by HelloLaohe** [comment]
在"图片后期"放大时，界面有一些不一样，不知道是什么问题

**Reply by HelloLaohe** [reply]
.unequal-height.svelte-15lo0d8{
    display: contents !important;
}

**Comment by chanry520** [comment]
win11系统，悬浮侧边栏失效，鼠标靠近浏览器边缘，侧边栏不会呼出

**Comment by caojingfei** [comment]
Bug反馈: 双列模式下, 文生图没有进度条, 也没有预览

**Comment by yesky216** [comment]
I love the new interface, you are a genius!

**Comment by jasoncow007** [comment]
发送到openoutpaint按钮失效，原UI下正常

**Comment by jasoncow007** [comment]
浮动失效了有人报了么？

**Comment by caojingfei** [comment]
双列模式下, 没有进度条也没有预览的bug，浮动失效bug，send outpaint失效bug，，，期待解决

**Comment by caojingfei** [comment]
双列模式下, 没有进度条也没有预览的bug，浮动失效bug，send outpaint失效bug，，，期待解决

**Comment by seventyer** [comment]
sdweb-easy-prompt-selector  这个插件 用咱们的主题以后 下拉菜单背景是白色的 字体也是白色的 根本看不清。

**Comment by X43dogeMan** [comment]
in dark mode, the font on the prompt is black/green. Can't see it unless highlighted. Also if the prompt has a lora error in the light theme. It will be red and you can't fix this. This is in the lobehud theme?

**Comment by suonnon** [comment]
I hope Text color of Prompt and Negative Prompt could be customizable!!! I don't like the pink color of the negative prompt text :(

**Comment by QL-boy** [comment]
After the text generates an image, the following image information will not follow the current view of the image to change, sent to other workspaces is only sent to a batch of the first picture, will not be sent to the current view of the image.

**Comment by Tilmirs** [comment]
I hope to customize the font thickness, but for me, the interface is currently very difficult to recognize unselected items in dark mode

**Comment by QL-boy** [comment]
A bug was found where the top navigation bar would cover the fixed panel of ComfyUI and make it inoperable.
https://github.com/ModelSurge/sd-webui-comfyui

**Comment by PixelCat55** [comment]
Changes made to the theme settings are ineffective as they fail to save.

**Comment by dotargz** [comment]
I love everything about this theme, but the styles drop down cant fit long names in it without overflowing now!

**Reply by suonnon** [reply]
Same comment! This should be change, need a better UI/UX solution

**Comment by Lwying6789** [comment]
如果每个框架，都可以随意移动布置位置，就更好了，感觉好多布局不是很好看！

**Comment by sofunmanager** [comment]
双列模式下，用了提示词美化插件（如：Sd Webui Prompt All In One），提示词过多时鼠标放到提示词上会出现左侧宽度疯狂跳动.，不知道是冲突了还是什么原因期待解决...

**Comment by murazi** [comment]
界面的颜色太黑了，眼睛看着非常难受，可以改色调吗，希望能够稍微量一点，对比不要那么强

**Comment by Olberg55** [comment]
https://gyazo.com/310bbb48022cb8e85b731621bb6e56c4 
These buttons are not obvious at all without the title.

**Reply by Olberg55** [reply]
https://gyazo.com/234de3a59556926367ecd6dad5ceb741 only the magic stick has title.

**Comment by Ataoya1** [comment]
生成下面的快捷按钮全部跑到主界面去了

**Comment by fredart-91** [comment]
hi, i used std mac (v 1.5.1) , ModuleNotFoundError: No module named 'insightface' and AttributeError: 'GeneralConditioner' object has no attribute 'process_tokens, who can help me,   all my LORA models are no longer displayed and extension  and extension ROOP does not work
and I can't change the color of the positive and negative prompt text (black on black) I can't see anything

**Comment by qianxingstudio** [comment]
底色的那个色彩流动还有毛玻璃的效果是否可以关闭

**Comment by anne43983959** [comment]
话说什么时候修正1.6RC版本两种手风琴菜单显示效果不一致的问题呢

**Comment by canisminor1990** [comment]
⚠️⚠️⚠️ Lobe Theme v3 is compatible with SD WebUI v1.6 and is not backwards compatible. Versions below < 1.6 should be moved to the branch [legacy-sd-webui-1.5](https://github.com/lobehub/sd-webui-lobe-theme/tree/legacy-sd-webui-1.5)

Currently, it can run normally with the default configuration, but there are still some minor styling issues that need to be fixed.

**Reply by Dragoy** [reply]
SD.NEXT support ever planned?

**Comment by cdnuoyan** [comment]
The menu bar on the left and right sides should be moved down without blocking the top horizontal menu. This will make the top horizontal menu bar display fully without being hidden, and can make the logo display smaller or hidden.

**Comment by kakachiex2** [comment]
I love cappuccino theme for webui can you make it compatible or there is a way to customize primary color to setup my own palette.
here is cappuccino webui theme: https://github.com/catppuccin/stable-diffusion-webui

**Reply by kakachiex2** [reply]
specially the latte theme is beautiful and easy to the eyes it would be good if you integrate it to lobehub
https://raw.githubusercontent.com/catppuccin/stable-diffusion-webui/main/assets/latte.webp

**Comment by jasoncow007** [comment]
1.6适配有问题呀，插件显示有问题，多了好多空行。。。还有个比较严重的就是貌似加载完了，但是模型lora都一直加载中，脚本也加载不了

**Reply by bugscats** [reply]
我也是这样的，你解决了吗？

**Comment by kakachiex2** [comment]
The Comfyui extension for webui, doesn't load with this theme not compatible but wen I' disable the lobehub theme the extension show up.

**Comment by Terry0712** [comment]
为什么文生图和图生图的插件界面里，Controlnet和Additional Networks标题之间的间距非常大呢？好像是多出来了一个空白的插件的距离，面Additional Networks和Tiled VAE之间的间距又是正常的

**Comment by paperwhite2023** [comment]
无法在离线的网路环境下使用，能否提供一个离线模式？

**Comment by anne43983959** [comment]
插件检查更新的时候格式有点崩啊

**Comment by JustinYChen** [comment]
点击右上角设置，没有东西

**Reply by JustinYChen** [reply]
f11全屏之后正常了

**Comment by widl** [comment]
模型列表加载完成后，依然留有加载标志

**Comment by mingmingyaozhidao** [comment]
Does anyone know why the right column of the Sd Webui Lobe Theme plug-in is empty and no model appears?

**Reply by liuyakun6** [reply]
嗯查出来了跟sd-webui-prompt-all-in-one这个插件冲突了，大佬可以修复一下吗？你们这俩个插件我都好喜欢啊，难以取舍，呜呜呜

**Comment by liuyakun6** [comment]
双列模式不起作用，开完了之后生成还是在下面

**Reply by liuyakun6** [reply]
好像还有报错？大佬帮我看看
[Lobe]: Initializing Lobe
Startup time: 11.5s (prepare environment: 2.3s, import torch: 2.2s, import gradio: 0.8s, setup paths: 0.4s, initialize shared: 0.3s, other imports: 0.4s, load scripts: 1.3s, create ui: 1.0s, gradio launch: 2.7s).
refresh_ui
Traceback (most recent call last):
  File "D:\AIGC\SD\py310\lib\site-packages\gradio\routes.py", line 488, in run_predict
    output = await app.get_blocks().process_api(
  File "D:\AIGC\SD\py310\lib\site-packages\gradio\blocks.py", line 1431, in process_api
    result = await self.call_function(
  File "D:\AIGC\SD\py310\lib\site-packages\gradio\blocks.py", line 1103, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "D:\AIGC\SD\py310\lib\site-packages\anyio\to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "D:\AIGC\SD\py310\lib\site-packages\anyio\_backends\_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "D:\AIGC\SD\py310\lib\site-packages\anyio\_backends\_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "D:\AIGC\SD\py310\lib\site-packages\gradio\utils.py", line 707, in wrapper
    response = f(*args, **kwargs)
  File "D:\AIGC\SD\modules\ui_extra_networks.py", line 392, in pages_html
    return refresh()
  File "D:\AIGC\SD\modules\ui_extra_networks.py", line 400, in refresh
    ui.pages_contents = [pg.create_html(ui.tabname) for pg in ui.stored_extra_pages]
  File "D:\AIGC\SD\modules\ui_extra_networks.py", line 400, in <listcomp>
    ui.pages_contents = [pg.create_html(ui.tabname) for pg in ui.stored_extra_pages]
  File "D:\AIGC\SD\modules\ui_extra_networks.py", line 162, in create_html
    self.items = {x["name"]: x for x in self.list_items()}
  File "D:\AIGC\SD\modules\ui_extra_networks.py", line 162, in <dictcomp>
    self.items = {x["name"]: x for x in self.list_items()}
  File "D:\AIGC\SD\modules\ui_extra_networks_textual_inversion.py", line 32, in list_items
    for index, name in enumerate(sd_hijack.model_hijack.embedding_db.word_embeddings):
RuntimeError: dictionary changed size during iteration
提示：Python 运行时抛出了一个异常。请检查疑难解答页面。

**Reply by canisminor1990** [reply]
sd webui 什么版本

**Reply by liuyakun6** [reply]
最新版的1.6.0

**Reply by canisminor1990** [reply]
是官方版本么，试试关闭其他插件，看看是不是插件冲突导致的

**Reply by liuyakun6** [reply]
是秋叶大佬的整合包，我先试试关掉其他插件

**Reply by liuyakun6** [reply]
应该是是冲突了，我看看具体是哪个

**Reply by liuyakun6** [reply]
嗯查出来了跟sd-webui-prompt-all-in-one这个插件冲突了，大佬可以修复一下吗？你们这俩个插件我都好喜欢啊，难以取舍，呜呜呜，（尴尬刚才回复错地方了，咋撤回啊？）

**Reply by liuyakun6** [reply]
这个是sd-webui-prompt-all-in-one插件的网址https://gitcode.net/ranting8323/sd-webui-prompt-all-in-one

**Comment by royshen007** [comment]
双列模式下，无法显示常用提示词选项
In dual-column mode, common prompt words cannot be displayed

**Comment by sw854638335** [comment]
Everything under the additional network is blank, and neither the model nor Lora has been transferred there.

**Comment by RJSprod** [comment]
Finally giving this UI a try after frustration with standard UI, i really love it so far... with one complaint.

There needs to be an easier way to open and collapse the extra networks panel. Mouse over is okay, but i really wish there was a BUTTON to open the panel that had a permanent home, same with quick settings.

This is my biggest complaint... another way to solve this is to make the mouse over area (that activates the panel) much much larger. It feels like its just a few pixels from the right side of the screen. I would definitely like to edit that and make it more than double the detection zone.

Otherwise, phenomenal UI!

**Comment by OCer** [comment]
顶部tabbar的双语插件失效了  https://github.com/journey-ad/sd-webui-bilingual-localization

**Comment by canisminor1990** [comment]
can't wait to share some new features with everyone. The latest version released today includes some new practical features, such as improved information display and sharing capabilities! I will add them to the document later.

![BFCA9684-DC21-463E-9CB8-1FC00D404308.png](https://github.com/lobehub/sd-webui-lobe-theme/assets/17870709/cce2f404-0fff-404c-8b0b-61f5177d203b)

![95D2EE34-472D-4ABC-ACAA-90E1E2BEB83B.png](https://github.com/lobehub/sd-webui-lobe-theme/assets/17870709/fee40aaf-fd46-4724-afd6-94979fe84569)

![LobeTheme_2023-11-28 (2).jpg](https://github.com/lobehub/sd-webui-lobe-theme/assets/17870709/4d56ba22-22f8-4032-9466-8a8dc8d44906)

**Comment by anne43983959** [comment]
打开生成信息替代以后，在重新加载过WebUI之后会失效，使得整个生成信息区域全部都消失

**Comment by makisekurisu-jp** [comment]
希望大大可以為ComfyUI做一個主題，可以將使用者介面做成類似於WebUI的版式，幫助轉向使用ComfyUI的新使用者快速上手。

**Reply by canisminor1990** [reply]
做过一个不过好久没维护了 https://github.com/canisminor1990/kitchen-comfyui

**Reply by makisekurisu-jp** [reply]
> 做过一个不过好久没维护了 https://github.com/canisminor1990/kitchen-comfyui

因為新技術的出現，比如
Latent Consistency Model，
stable video diffusion，
sdxl turbo等，
ComfyUI成為目前非常有效體驗新技術的使用者介面。

**Comment by UN-xing** [comment]
模型一直加载中

**Reply by canisminor1990** [reply]
尝试关闭 gradio 队列功能

**Comment by kakachiex2** [comment]
Can you add option to use user.css as a color theme, I' don't like the default color theme of lebehub and when I'm not connected to the internet the webui don't load it took forever loading looks like your theme is loading something from the net.

**Comment by xixibincn** [comment]
每次打开模型区域都无法及时打开，总的刷新几次才会显示正常，谢谢作者辛苦开发，希望以后会有改善

**Comment by xixibincn** [comment]
扩展模型 无法加载很慢很困难，希望可以把这个bug修复谢谢！

**Comment by search620** [comment]
amazing amazing amazing!!! the best skin ever made for stable diffusion!!!!
can I make the pin to be hidden by default? I mean that when I open the UI then it will be hidden automatically without me to configure it at each run?

**Comment by ximuhai** [comment]
分割预览失效了，生成的图片还是会随着左边页面滚动

**Reply by canisminor1990** [reply]
webui版本多少

**Comment by 3020296324** [comment]
右侧预览会随机隐藏部分模型和`LORA刷新也不显示.

**Comment by kakachiex2** [comment]
Extra network tab don't load on Webui Forge I will write bug in forge Branch

**Comment by seedhunterai** [comment]
thanks for making. Prompt editor is useful. Just how should have been implemented..one feature that is useful is a plugin that the user set  an aesthetics value and it will not save the image unless met. Seems useful.

**Comment by ahem95** [comment]
There is a bug that the quick settings are constantly loading

**Comment by guodeying** [comment]
为什么安装了inpaint anything 看不到！！！
请检查确认。谢谢

**Comment by man460** [comment]
不翻墙的情况的下，页面启动会有调用接口的请求会报错。然后页面会一直卡在初始化页面的状态下。然后原本页面会加载在其下面。导致布局异常
https://api.github.com/repos/lobehub/sd-webui-lobe-theme/releases/latest
https://api.github.com/repos/zanllp/sd-webui-infinite-image-browsing/commits?per_page=1
https://api.github.com/repos/zanllp/sd-webui-infinite-image-browsing/releases/latest
能否停止调用这三个接口，或者改成本地文件。
还有说需要修改我的网络配置？

**Comment by realmalikov** [comment]
I don't know why the 16px padding is missing here in Forge, could you please add it?
![grafik](https://github.com/user-attachments/assets/c132ee5c-20c9-4298-9a3e-59758f9fbf09)

**Comment by ApproLight01** [comment]
<img width="942" alt="1737364050151" src="https://github.com/user-attachments/assets/42e4ecb1-07e8-4168-9a5a-41a3cdfde25d" />
how can I change the picture and text in loading page?I cant found these features in the theme config.json or any other files.

**Comment by LindezaBlue** [comment]
I noticed changes to the code was 9 months ago is this still being actively developed?
![{2A6ABA71-62B8-48CF-9775-19D2E00A007E}](https://github.com/user-attachments/assets/fcd5bf8d-d74b-4412-a3a6-2d73cc1a3647)

**Reply by ErraticFox** [reply]
Did you get this to work? I was able to finally "enable" it with the modern theme type selected, but it doesn't work and console only outputs "🤯 LobeTheme: Initializing..."

**Comment by raffaeleg4** [comment]
Beautiful interface! I don't see Italian translation yet. In case you want to add, let me know!

**Comment by y7643982-star** [comment]
涂鸦重绘没办法调用吸取颜色按钮

**Comment by jqliux** [comment]
谷歌浏览器ver139.0.7258.67，左侧栏的快捷设置样式乱了，怎么调整

**Comment by suqiyue27** [comment]
出大BUG了，TI嵌入扫描不出来（呜呜呜）
明明就在embeddings里面，可就是暂无内容
